coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java

```diff
@@ -36,7 +36,7 @@
     
     public static final String BROWSER_PROTOCOL = "http";
     public static final String BROWSER_ADDRESS = "127.0.0.1";
-    public static final String PORTAL_ADDRESS = "kc-dev";
+    public static final String PORTAL_ADDRESS = "";
```

Comment:
@leoherbie can you look at this change? This is weird to me. Why do we even have this value if it isn't necessary?